### ISSUE TITLE
Column size does not account for visual length of caracters

### ISSUE THREAD

**Issue by Kasama** [MaxTaggart/columnar#6]
First of all, thanks for the project, it really makes life a bit easier.

I'm facing a problem that whenever the characters inside a table cell span multiple terminal cells (such as emojis, japanese and other unicode characters), the width of the cell counts it as only one character.

The problem here is probably the use of `len` to determine the length of the column, which works fine for most eastern characters. I'd suggest using something along the lines of https://github.com/jquast/wcwidth to find the visual column width.

Small example:
```python
from columnar import columnar
data = ["👍", "another column"]
headers = ["A", "B"]
table = columnar(data, headers)
print(table)
```
and its output
```
|-----|--------------|
|A    |B             |
|====================|
|👍    |another column|
|-----|--------------|
```

**Comment by acksmaggart** [comment]
Hey @Kasama, sorry for the wait and thanks for pointing that out! That's an interesting one. In addition to using something like wcwidth we will probably have to add a little extra logic for wrapping cells that contain characters that are wider than 1 column. I'll look into it.

**Comment by acksmaggart** [comment]
So I did a little re-factoring and added support for double-width characters incorporating the `wcwidth` library in PR #7 . Things are looking good in testing but let me know if it is still giving you trouble. Support for double-wide characters will be included in version 1.3.0 which I will hopefully have up on pypi later tonight.
